context.go

```diff
@@ -23,6 +40,16 @@ func newContext(c *Conode, o *Overlay, servID ServiceID, manager *serviceManager
 	}
 }
 
+// contextDataPath indicates where the service-data will be stored. If it is
+// empty, a memory-map is used.
+var contextDataPath = ""
```

Comment:
Just for the record, I'm ok now with this custom rule of being empty now, but we'll have to change that once we have a config file / struct in place. Like a real interface `Memory`, `File`, `UploadAmazon` whatever. :D